source/index.html.md

```diff
@@ -1023,8 +1027,16 @@ A successful request returns the HTTP `200 OK` status code with the following JS
 | Parameter | Type | Example | Description |
 | ---| ---| ---| --- |
 | amount | Money | 100.00 | The user's balance to a maximum of 2 decimal places. |
-| currency | string | USD | Currency specified by an ISO-4217 3-letter code. |
+| currency | string | USD | Currency.  When relevant this will reference ISO-4217 3-letter code. |
+
+
+<aside class="comment">
+Currency code <code>XXX</code> will be used to represent free money funds.
+</aside>
 
+<aside class="comment">
+Currencies are associated to accounts during creation of the integration.
```

Comment:
"creation of the integration" doesn't make sense to me and might need rewording.

Reply:
```suggestion
Possible currencies returned are setup within the Betable API during the initial phases of partner integration
```